src/open_dread_rando/patcher_editor.py

```diff
@@ -120,27 +136,36 @@ def copy_actor(self, scenario: str, coords, templateActor: Container, newName: s
         newActor = copy.deepcopy(templateActor)
         newActor.sName = newName
         currentScenario = self.get_scenario(scenario)
-        currentScenario.actors_for_layer('default')[newActor.sName] = newActor
+        currentScenario.actors_for_sublayer("default")[newActor.sName] = newActor
         newActor.vPos = [float(c) + offset for c, offset in zip(coords, offset)]
 
         return newActor
 
-    def find_type_of_actor(self, scenarioStr: str, actordef: str, layer: str = "default"):
+    def find_type_of_actor(self, scenario_name: str, actordef: str,
+                           actor_layer: ActorLayer = ActorLayer.ENTITIES) -> list[tuple[str, Container]]:
```

Comment:
Maybe this should return a dict? Not sure if there's any reason to have this as a list.

Reply:
either way this needs to be updated since it's returning `list[tuple[str, str, Container]]`

Reply:
done